examples/basic/annotations/legends_item_visibility.py

```diff
@@ -1,3 +1,11 @@
+''' Marker and line plots that demonstrate manual control of legend visibility of individual items
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.circle, bokeh.plotting.figure.line
```

Comment:
This uses the LegendItem object, at least behind the scenes, so we could add that here as well:

```suggestion
    :apis: bokeh.models.LegendItem, bokeh.plotting.figure.circle, bokeh.plotting.figure.line
```